var/spack/repos/builtin/packages/py-altair/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyAltair(PythonPackage):
+    """Declarative statistical visualization library for Python"""
+
+    homepage = "https://altair-viz.github.io/"
+    url = "https://github.com/altair-viz/altair/archive/refs/tags/v4.2.0.tar.gz"
```

Comment:
If possible we prefer to use PyPI tarballs instead of GitHub tarballs: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github